src/dapps/preview/preview-options.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  PreviewCamera,
+  PreviewEmote,
+  PreviewEnv,
+  WearableBodyShape
+} from '../..'
+
+/** @alpha */
+export type PreviewOptions = {
+  contractAddress?: string | null
```

Comment:
Just of curiosity, is there any reason why these properties can be `undefined | null | string`?

Reply:
convenience, having them supported as both null and undefined make it easier to configure via query params (they are `null` if not found) or programatically (like when configuring via postMessage)